tests/util/version.js

```diff
@@ -79,6 +79,22 @@ describe('Version', () => {
         ['Warning: Flow version was set to "detect" in eslint-plugin-react settings, but the "flow-bin" package is not installed. Assuming latest Flow version for linting.']
       ];
     });
+
+    it('works with virtual filename', () => {
+      sinon.stub(context, 'getFilename').callsFake(() => path.resolve(base, 'detect-version-sibling', 'test.js/0_fake.js'));
```

Comment:
there's nothing about this filename that actually ensures it's "virtual" - this could absolutely be a real file path.

Reply:
No, a filename can never be a directory.

`test.js` a true filename, there will never be a `test.js/foo` file.

Reply:
Yes, and the test here `test.js` is a true file.